pyproject.toml

```
@@ -0,0 +1,11 @@
+[build-system]
+  requires = ["setuptools"]
+  build-backend = "setuptools.build_meta"
+
+[project]
+  name = "penelope"
+  version = "0.9.2"
+  requires-python = ">=3.6"
+
```

If you add a `main()` function to penelope.py and invoke it in `if __name__ == "__main__":` then you can also make pip/setuptools install a shell entrypoint for you upon install:

```suggestion
[project.scripts]
penelope = "penelope:main"
```

---

I added the main() function. Thanks!